src/pkg/cli/composeUp.go

```diff
@@ -49,7 +39,7 @@ func ComposeUp(ctx context.Context, c client.Client, force bool, mode defangv1.D
 		return nil, project, &ComposeError{fmt.Errorf("no services found")}
 	}
 
-	if DoDryRun {
+	if force == compose.BuildContextIgnore {
```

Comment:
Should we rename the variable `force`? it is a bit confusing

Reply:
good call